ext/hal/nxp/mcux/README

```diff
@@ -24,6 +24,9 @@ Status:
    MIMXRT1021	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
    MIMXRT1064	SDK 2.4.0 (2018-10-22)		REL_SDK_2.4.0_RT1064_RFP_INTERNAL
    MK64F12	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
+   MKE14F16	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
+   MKE16F16	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
+   MKE18F16	SDK 2.5.0 (2018-12-17)		REL_2.5.0_REL9_RFP_RC3_7_1
```

Comment:
@MaureenHelm I am not sure if the tag is correct for these imports. Can I see the tag from the generated MCUX SDK archive somehow?

Reply:
<img width="861" alt="Screenshot 2019-05-10 at 19 37 26" src="https://user-images.githubusercontent.com/1076226/57545815-bc4ea680-735b-11e9-81bd-15f7809db89d.png">

@MaureenHelm My list of recent SDKs does not mention the tag. Perhaps I should leave them blank (or `-` in the `README` file?

Reply:
Let me check with the SDK team to find out why the tag isn't there

Reply:
It turns out that only employee accounts can see the tag. Instead, please use the "revision" attribute from `TWR-KE18F_manifest_v3_4.xml`:
```xml
<ksdk id="MCUXpresso250" name="MCUXpresso250" version="2.5.0" revision="150 2018-12-17"/>
```

Reply:
@MaureenHelm Thank you for following up on this. I have changed the `README` file to list the `revision` attribute values.